emsdk.bat

```diff
@@ -34,8 +34,9 @@ set EMSDK_PY=python
 
 :end
 call %EMSDK_PY% "%~dp0\emsdk.py" %*
+set _EMSDK_ERRORLEVEL=%ERRORLEVEL%
 
-endlocal
+endlocal & set "_EMSDK_ERRORLEVEL=%_EMSDK_ERRORLEVEL%"

```

Comment:
Shouldn't we instead early exit if this command fails?

We don't want to go on run the other stuff here if the emsdk command fail.

Reply:
Good point. That is logically better, though doesn't really matter in practice either way, unlikely we'll generate the emsdk_set_env.bat file any way.

But it does give simpler code since it allows avoiding leaking that _EMSDK_ERRORLEVEL env. var. outside, so migrated to that code.